Makefile

```diff
@@ -279,7 +279,7 @@ docker-metrics-test:     ## Run metrics helper unit test suite in a container.
 		make metrics-unit-test
 
 # Fetch the CNI plugins
-plugins: FETCH_VERSION=1.4.0
```

Comment:
We have to look at the change log to see if these introduce any backwards incompatiblity changes. We had revert it back to 1.4.x sometime ago, if I remember correctly. 

cc: @M00nF1sh  do you recollect?

Reply:
Diff
https://github.com/containernetworking/plugins/compare/v1.4.0...v1.5.1

Reply:
This update is fine. We aren't changing the CNI spec here.